util/other.syms

```diff
@@ -834,3 +834,4 @@ EVP_PKEY_base_id                        define
 SSL_set_retry_verify                    define
 TS_VERIFY_CTX                           define
 CMAC_CTX                                define
+SSL_get_builtin_sigalgs                 define
```

Comment:
This line should be removed as it is not a define

Reply:
done